pkg/podvolumebackup/util.go

```diff
@@ -0,0 +1,20 @@
+package podvolumebackup
```

Comment:
The two UpdateStatusToFailed functions work for podvolumebackup/podvolumerestore CR only, they don't look like common functions that could be used by others in future. Therefore, I suggest we still put the two functions in the controller files. This will also make the code here compliance with main and 1.10 branch as the comment[ here](https://github.com/vmware-tanzu/velero/pull/5830/files#r1098080167)